spinnaker/api/models.go

```diff
@@ -0,0 +1,33 @@
+package api
+
+// PipelineTemplateV2 defines the schema of the pipeline template JSON.
+type PipelineTemplateV2 struct {
+	ID        string                       `json:"id,omitempty"`
+	Metadata  PipelineTemplateV2Metadata   `json:"metadata"`
+	Pipeline  interface{}                  `json:"pipeline"`
+	Protect   *bool                        `json:"protect,omitempty"`
+	Schema    string                       `json:"schema"`
+	Variables []PipelineTemplateV2Variable `json:"variables,omitempty"`
+}
+
+type PipelineTemplateV2Metadata struct {
+	Name        string   `json:"name"`
+	Description string   `json:"description"`
+	Owner       *string  `json:"owner,omitempty"`
+	Scopes      []string `json:"scopes,omitempty"`
+}
+
+type PipelineTemplateV2Variable struct {
+	Name         string      `json:"name"`
+	Description  *string     `json:"description,omitempty"`
+	DefaultValue interface{} `json:"defaultValue,omitempty"`
+	Type         string      `json:"type"`
+	Group        *string     `json:"group,omitempty"`
+	Example      *string     `json:"example,omitempty"`
```

Comment:
I cannot see Group and Example anymore at the V2 spec (https://spinnaker.io/reference/pipeline/templates/) but only at V1 (https://github.com/spinnaker/dcd-spec/blob/master/PIPELINE_TEMPLATES.md#variables). Are they still needed for V2?

Reply:
You are right. I got confused by the last line on the page and while looking at the V1 docs i thought these fields are also available in V2. I can remove them.